package.json

```diff
@@ -64,13 +67,15 @@
     "big-integer": "^1.6.41",
     "buffer": "^5.2.1",
     "core-js": "^2.6.4",
+    "cypress": "^3.1.5",
```

Comment:
this is still making netlify fail as mentioned here: #275
See related PR: https://github.com/xlayers/xlayers/commit/b9a2676ab51301b7eb510167face85f3682e22de

Any idea how this can be fixed?

Reply:
I will take a look later today after work

Reply:
hey @krvajal after thinking about this. You can leave the cypress dep here. We will update netlify and make just build the app without running e2e tests.